### PR TITLE
Upgrade to Ember 2.11.1

### PR DESCRIPTION
**Pull request by toddjordan**
#136

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,26 +52,28 @@
...
     "ember-export-application-global": "^1.0.5",
     "ember-font-awesome": "2.2.0",
     "ember-inflector": "^1.9.4",
-    "ember-load-initializers": "^0.5.1",
+    "ember-load-initializers": "^0.6.0",
     "ember-lodash": "0.0.11",
     "ember-pouch": "github:gaurav0/ember-pouch#c89600c7b8c14c919b9b808a4ea6d9d6666ca047",
     "ember-resolver": "^2.0.3",
     "ember-route-action-helper": "2.0.0",
+    "ember-source": "~2.11.0",
```
Comment (Gaurav0) [inline package.json]: Can we make this 2.11.1?

File: tests/helpers/start-app.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ import config from '../../config/environment';
 export default function startApp(attrs) {
   let application;
 
-  // use defaults, but you can override
-  let attributes = Ember.assign({}, config.APP, attrs);
+  let attributes = Ember.merge({}, config.APP);
+  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
```
Comment (Gaurav0) [inline tests/helpers/start-app.js]: This revert is only needed in addons for compat with Ember 2.4. As this is an app, lets keep the assign code.

File: package.json (hunk 1 above)
Comment (locks) [inline package.json]: 2.11.2 is out with the missing fixtures fix, fyi

Comment (toddjordan) [inline package.json]: I'll up the version 👍

Comment (toddjordan) [inline package.json]: though it should be picking up patch releases with ~

File: package.json [hunk 3]
```diff
@@ -52,26 +52,28 @@
...
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-qunit": "^3.0.1",
     "ember-cli-release": "^0.2.9",
     "ember-cli-sass": "^5.6.0",
+    "ember-cli-shims": "^1.0.2",
     "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
     "ember-composable-helpers": "^0.20.0",
-    "ember-data": "~2.10.0",
+    "ember-data": "^2.11.0",
```
Comment (Gaurav0) [inline package.json]: I think we should stick with ~ here.

File: package.json [hunk 4]
```diff
@@ -40,7 +40,7 @@
     "ember-ajax": "^2.4.1",
     "ember-anchor": "~0.1.8",
     "ember-browserify": "~1.1.13",
-    "ember-cli": "2.10.0",
+    "ember-cli": "^2.11.1",
```
Comment (Gaurav0) [inline package.json]: No ^ here. We want ember-cli updates to take place manually.

File: package.json (hunk 3 above)
Comment (toddjordan) [inline package.json]: Got this from here: https://github.com/ember-cli/ember-new-output/compare/v2.10.1...v2.11.1

Comment (toddjordan) [inline package.json]: Maybe because ember-data is more semver friendly?

File: package.json (hunk 4 above)
Comment (toddjordan) [inline package.json]: yeah, originally I had ~ which was fine.  Didn't really even need to make it 2.11.1 (because the ~).  Will switch back

**Pull request merged**